transport_test.go

```diff
@@ -0,0 +1,141 @@
+package lumigotracer
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
```

Comment:
Don't we need to import function from `go-tracer-beta`?

Reply:
nope it's in the same package